control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/files/infra_role.yaml

```diff
@@ -8,10 +8,10 @@ rules:
   resources: ["datavolumes"]
   verbs: ["get", "create", "delete"]
 - apiGroups: ["kubevirt.io"]
-  resources: ["virtualmachineinstances"]
+  resources: ["virtualmachineinstances", "virtualmachines"]
   verbs: ["list", "get"]
 - apiGroups: ["subresources.kubevirt.io"]
-  resources: ["virtualmachineinstances/addvolume", "virtualmachineinstances/removevolume"]
+  resources: ["virtualmachineinstances/addvolume", "virtualmachineinstances/removevolume", "virtualmachines/addvolume", "virtualmachines/removevolume"]
   verbs: ["update"]
```

Comment:
I believe we lost the newline in the backport?

Reply:
Are you referring to the newline at the end of the file? IMO, I don't think it makes much of a difference.